river/ensemble/adaptive_random_forest.py

```diff
@@ -399,7 +399,7 @@ class AdaptiveRandomForestClassifier(BaseForest, base.Classifier):
     >>> metric = metrics.Accuracy()
 
     >>> evaluate.progressive_val_score(dataset, model, metric)
-    Accuracy: 68.17%
+    Accuracy: 72.87%
```

Comment:
I assume this comes from the change in the normalization.

Reply:
I think so! Good news :D